app/models/agents/local_file_agent.rb

```diff
@@ -0,0 +1,186 @@
+module Agents
+  class LocalFileAgent < Agent
+    include LongRunnable
+    include FormConfigurable
+    include FileHandling
+
+    emits_file_pointer!
+
+    default_schedule 'every_1h'
+
+    def self.should_run?
+      ENV['ENABLE_INSECURE_AGENTS'] == "true"
+    end
+
+    description do
+      <<-MD
+        The LocalFileAgent can watch a file/directory for changes or emit an event for every file in that directory. When receiving an event it writes the received data into a file.
+
+        `mode` determines if the agent is emitting events for (changed) files or writing received event data to disk.
+
+        ### Reading
+
+        Then `watch` is set to `true` the LocalFileAgent will watch the specified `path` for changes. An event will be emitted for every detected change.
+
+        Then `watch` is set to `false` the agent will emit an event for every file in the directory on each sheduled run.
+
+        #{emitting_file_handling_agent_description}
+
+        ### Writing
+
+        Every event will be writting into a file at `path`, Liquid interpolation is possible to change the path per event.
+
+        When `append` is true the received data will be appended to the file.
+
+        Use [JSONPath](http://goessner.net/articles/JsonPath/) to select the which part of the received event should be written in `data_path`.
+
+        *Warning*: This type of Agent can read and write any file the user that runs the Huginn server has access to, and is #{Agents::LocalFileAgent.should_run? ? "**currently enabled**" : "**currently disabled**"}.
+        Only enable this Agent if you trust everyone using your Huginn installation.
+        You can enable this Agent in your .env file by setting `ENABLE_INSECURE_AGENTS` to `true`.
```

Comment:
The `HUGINN_` prefix is still supported for the multi-process image, but it's not required anymore. The normal env variable names work as well.